README.md

```diff
@@ -1,30 +1,65 @@
-# my-component
+# Storage Extractor
 
-[![Build Status](https://travis-ci.org/keboola/my-component.svg?branch=master)](https://travis-ci.org/keboola/my-component)
+[![Build Status](https://travis-ci.org/keboola/ex-storage.svg?branch=master)](https://travis-ci.org/keboola/ex-storage)
 
-> Fill in description
+Extracts tables from source bucket from a source project. You need to provide a Storage
+token (which identifies both the project and the bucket) from the source project which 
+has `read` access to the source bucket **only**.
 
 # Usage
 
-> fill in usage instructions
+Configuration:
+
+```
+{
+	"parameters": {
+		"#token": "some-token",
+		"url": "https://connection.keboola.com/",
+		"tableName": "some-table"
+	}
+}
+```
+
+For incremental loading, you the `changedSince` parameter can be provided:
+
+```
+{
+	"parameters": {
+		"#token": "some-token",
+		"url": "https://connection.keboola.com/",
+		"tableName": "some-table",
```

Comment:
nestačilo by `table`?

Reply:
nestacilo, protoze pak to svadi k tomu zadat tableId jako identifikator tabulky

Reply:
jasně